### ISSUE TITLE
Unity 2018 OSX LibPdFilterRead.cs errors

### ISSUE THREAD

**Issue by MacroMachines** [patricksebastien/libpd4unity#27]
opening the demo project in unity 2018.1.11 on OSX 10.12
Initially got several errors to turn on Player settings / allow unsafe, after doing this, the demo scene project still gets 2 erorrs that appear to be related to 16 bit short:

Assets/LibPdFilterRead.cs(52,10): error CS1502: The best overloaded method match for `LibPDBinding.LibPD.Process(int, short[], short[])' has some invalid arguments

Assets/LibPdFilterRead.cs(52,33): error CS1503: Argument `#2' cannot convert `System.IntPtr' expression to type `short[]'

**Comment by owenitt87** [comment]
I know this thread is a little dated now but im having this issue too. Dont suppose this got solved at all did it?

**Comment by patricksebastien** [comment]
Nope, I think it's better to use Heavy to compile your pd patch to C# / Unity:
https://github.com/enzienaudio/hvcc
